docs/local_testnet_deployment_plan.md

```diff
@@ -375,13 +370,13 @@ echo "=== set contract to eosio.token ==="
 #### create, issue & transfer tokens
 ```shell
 ./cleos push action eosio.token create '{"issuer":"eosio", "maximum_supply":"1000000000.0000 SYS", "can_freeze":0, "can_recall":0, "can_whitelist":0}' -p eosio.token@active
-./cleos push action eosio.token create '{"issuer":"eosio", "maximum_supply":"1000000000.0000 EOS", "can_freeze":0, "can_recall":0, "can_whitelist":0}' -p eosio.token@active
+./cleos push action eosio.token create '{"issuer":"eosio", "maximum_supply":"1000000000.0000 A", "can_freeze":0, "can_recall":0, "can_whitelist":0}' -p eosio.token@active
```

Comment:
This change won't work. The `eosio.token` contract does not manage the `A` token; the `core.vaulta` conract does. These instructions should either be updated to show how to setup these contracts and the `core.vaulta` wrapper contract. Or we keep it simpler and just stick to using `EOS` for these local deployments to avoid the extra complexity.